ABOUT.md

```diff
@@ -105,24 +108,3 @@ non-device-specific optimizations:
     with nGraph.
 -   **Memory management** -- Prevent peak memory usage by intercepting
     a graph with or by a "saved checkpoint," and to enable data auditing.
-
-Beta Limitations
-----------------
-
-In this Beta release, nGraph only supports Just In Time compilation,
-but we plan to add support for Ahead of Time compilation in the official
-release of nGraph. nGraph currently has limited support for dynamic graphs.
-
-Current nGraph Compiler full stack
-----------------------------------
-
-![](doc/sphinx/source/graphics/arch_complex.png)
```

Comment:
@bdalby This is the outdated illustration that needs replaced if ONNXIFI is being removed: https://github.com/NervanaSystems/ngraph/blob/master/doc/sphinx/source/graphics/arch_complex.png